docs/03/getting-started-1.md

```diff
@@ -26,9 +26,28 @@ function App() {
 export default App
 ```
 
-The `baseUrl` property is typically configured with the address of a backend proxy. The recommended approach is to use a backend service that acquires OAuth2 tokens from the Gusto Embedded API for authenticated users and proxies API calls using those tokens. Learn more about configurintg this and setting up authentication in the `Authentication` section.
+The `baseUrl` property is configured with the address of your backend proxy which is detailed further in the following section.
 
-For more configurations available on the GustoApiProvider, see the docs in the `Styles and Theming` and `Translation` sections.
+## Configuring a backend proxy
+
+When building with the React SDK, a backend proxy is required. React SDK components do not make calls to the Gusto Embedded API directly. Instead, the `baseUrl` configuration defines the URL of your proxy server. This proxy layer gives you complete control over requests sent to Gusto, which is essential for:
+
+1. Authentication
+2. Providing the user IP address for form signing operations
+
+The React SDK is designed to mirror the [Gusto Embedded API Reference](https://docs.gusto.com/embedded-payroll/reference/whats-new-in-v2024-04-01) with a 1:1 mapping of endpoints. The SDK maintains consistent naming conventions, parameters, and response structures with the Gusto API.
+
+Your proxy server simply needs to forward incoming SDK requests to the corresponding Gusto API endpoints. No special endpoint mapping or request transformation is required, as the SDK requests already match the Gusto API format. The proxy's main task is adding the necessary authentication headers before forwarding the request.
+
+### Using the proxy for authentication
+
+The proxy layer allows for authentication. The recommended approach is to use a backend service that acquires OAuth2 tokens from the Gusto Embedded API for authenticated users and proxies API calls using those tokens. Learn more about configuring this and setting up authentication in the `Authentication` section.
+
+### Using the proxy to provide the user IP address
+
+Some UI workflows require users to sign forms, which need the user's IP address for security purposes. Since the SDK runs in the browser, it cannot reliably access the IP address.
```

Comment:
I think it might be worth calling out more that the client could spoof the IP instead of the lack of the browsers ability to get the IP. Because I think there are many ways to get the IP but none of them provide the IP in a way which can't be spoofed.

Reply:
Good callout! updated the wording, lmk if that accounts for the above

Reply:
Looks good!